python/feldera/output_handler.py

```diff
@@ -48,3 +48,13 @@ def to_pandas(self, clear_buffer: bool = True):
             self.buffer.clear()
 
         return res
+
+    def to_dict(self, clear_buffer: bool = True):
```

Comment:
Would it be better to make `clear` a separate method?

Reply:
I felt like that introduces more friction than necessary. In most cases, we likely want to clear the buffer anyway.